client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -192,6 +194,8 @@ export default class UpdateChecks extends PureComponent {
 
     const updateCheckInfo = await config.get(UPDATE_CHECKS_CONFIG_KEY);
 
+    updateCheckInfo.stagedRollout = true;
```

Comment:
Further `updateCheckInfo && updateCheckInfo.someProp` code suggests this can be non-existent. In that case, the line above will throw.

Reply:
![image](https://user-images.githubusercontent.com/28307541/129852887-06c78ff5-6a59-4f90-b3f2-e53bdc05869e.png)
it fails for a fresh start

Reply:
Thank you for catching this - the suggested change of course makes sense.